integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -309,13 +343,13 @@ private void verifyTenant(String address, String... expectedTopics) throws Excep
         }
     }
 
-    private void consumeAndVerify(String address, String topicName, String expectedKey, String expectedValue, boolean offsetCommit) {
-        consumeAndVerify(address, topicName, new LinkedList<>(List.of(matchesRecord(topicName, expectedKey, expectedValue))), offsetCommit);
+    private void consumeAndVerify(String address, String topicName, String groupName, String expectedKey, String expectedValue, boolean offsetCommit) {
+        consumeAndVerify(address, topicName, groupName, new LinkedList<>(List.of(matchesRecord(topicName, expectedKey, expectedValue))), offsetCommit);
```

Comment:
Should it be `groupName` or `groupId`?

In line 350 you have added a parameter into the `consumeAndVerify` method called `groupId` but this one is `groupName`. I think this should be consistent.

It should apply to all calls to that method.

Reply:
@franvila good catch. we should be careful to adhere to the naming conventions established by Apache Kafka.  It uses `group.id` in the client config and mostly `GroupId` in the RPC definitions, so +1 to the suggest to use `groupId` instead of `groupName` in our code.

Reply:
I've changed that now! Thanks